client/state/imports/reducer.js

```diff
@@ -0,0 +1,11 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import importerLocks from './importer-locks/reducer';
+import { combineReducers } from 'state/utils';
+
+export default combineReducers( {
+	importerLocks,
+} );
```

Comment:
I'm still not very keen on this state structure. There are two ways I imagine us examine import information: by site id and by import id. If I want some information about an import I think I would like to grab it all at once…

```js
state.imports[ importId ].isLocked
```

Another level may or may not be helpful…

```js
state.imports[ siteId ][ importId ].isLocked
```

Either way I'm leery about this…

```js
state.imports.importerLocks[ importId ]
state.imports.authorMappings[ importId ]
state.imports.steps[ importId ]
state.imports.bagels[ importId ]
```

Do you have compelling reasons to split the individual pieces of the importer state object verses storing them as a single atom? (Note that the reducers can combine to build that atom while remaining under one top-level state id).